reindex/reindex.go

```diff
@@ -0,0 +1,61 @@
+package reindex
```

Comment:
are there tests for this somewhere

Reply:
yes there are component tests for this, which test the POST /jobs endpoint

Reply:
ok